website/templates/project/contributors.mako

```diff
@@ -8,245 +8,365 @@
-                        <tbody data-bind="template: {
-                                name: 'contribTpl',
-                                foreach: adminContributors,
-                                as: 'contributor'
-                            }">
-                        </tbody>
-                    </table>
-                </div>
-                ${buttonGroup()}
+<div class="col-md-3 col-xs-12">
+    <div class="filters">
+        <input type="text" class="form-control searchable" id="nameSearch" placeholder="Filter by name"/>
+        <h5 class="m-t-md">Permissions
+                <i class="fa fa-question-circle permission-info"
+                    data-toggle="popover"
+                    data-title="Permission Information"
+                    data-container="body"
+                    data-placement="right"
+                    data-html="true"
+                ></i></h5>
+        <div class="btn-group btn-group-justified-vertical filtergroup" id='permissionFilter'>
+            <div class="btn-group">
+                <button class="filter-btn btn-default btn" id="admins">Administrator</button>
+            </div>
+            <div class="btn-group">
+                <button class="filter-btn btn-default btn" id="write">Read + Write</button>
             </div>
+            <div class="btn-group">
+                <button class="filter-btn btn-default btn" id="read">Read</button>
+            </div>
+        </div>
+        <h5 class="m-t-md">Bibliographic Contributor
+                <i class="fa fa-question-circle visibility-info"
+                    data-toggle="popover"
+                    data-title="Bibliographic Contributor Information"
+                    data-container="body"
+                    data-placement="right"
+                    data-html="true"
+                ></i></h5>
+        <div class="btn-group btn-group-justified-vertical filtergroup" id='visibleFilter'>
+            <div class="btn-group">
+                <button class="filter-btn btn-default btn" id='visible'>Bibliographic</button>
+            </div>
+            <div class="btn-group">
+                <button class="filter-btn btn-default btn" id='notVisible'>Non-Bibliographic</button>
+            </div>
+        </div>
+    </div>
+</div>
 
+<div class="col-md-9 col-xs-12">
+    <div id="manageContributors" class="scripted">
+        <h3> Contributors
+            <!-- ko if: canEdit -->
+                <a href="#addContributors" data-toggle="modal" class="btn btn-success btn-sm m-l-md">
+                  <i class="fa fa-plus"></i> Add
+                </a>
+            <!-- /ko -->
+        </h3>
+        % if 'admin' in user['permissions'] and not node['is_registration']:
+            <p class="m-b-xs">Drag and drop contributors to change listing order.</p>
+        % endif
+
+    <div data-bind="filters: {
+            items: ['.contrib', '.admin'],
+            toggleClass: 'btn-default btn-primary',
```

Comment:
IIRC, `btn-default` and `btn-primary` are mutually exclusive. Are you sure `btn-default` is supposed to be here?